src/data.py

```diff
@@ -261,6 +268,22 @@ def parse_static_op_hours(hours_list, eatery_id, dining_items):
     )
   return new_operating_hours
 
+def is_closed(current_date, dates_closed):
+  """Returns boolean indicating if the specified date is present in a list of closed dates.
+  Parameter current_date: a datetime.date object of the date to compare
+  Parameter dates_closed: a list of strings either in form 'mm/dd/yy' for a singular date or
+  'mm/dd/yy-mm/dd/yy' for a range of dates.
+  """
+  for dates in dates_closed:
```

Comment:
In terms of writing functions this isn't really good practice –– you want your helper functions to work independently.

For example, we can think about a different way to spec this function:

```python
def is_date_in_range(date_to_compare, start_date, end_date):
  """ Returns True if [date_to_compare] is between [start_date] and [end_date]
  All params are datetime objects.
  """
  return date_to_compare > start_date and date_to_compare < end_date
```

Much cleaner, right? Clearly, the obvious counterargument is that this isn't how your data is formatted: but there's only one usage of this function, and you can add a little bit of preprocessing, saving a lot of code.

In general, I think a majority of the functions in this file can have the same concepts applied, and it'll be much more readable for people in the future!

Reply:
Okay so you want less in the helper and then do the preprocessing in parse_static_op_hours()?

Reply:
+1 to young and also the more atomic and modular your functions are, the more useful they'll be if you ever need them again in the future
in general, refactoring more code out isn't necessarily the proper approach -- sometimes doing some preprocessing is actually easier to understand